Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN LC_ALL=C DEBIAN_FRONTEND=noninteractive apt-get install -qqy git cmake \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/partial/* /tmp/* /var/tmp/*
 
-RUN mkdir /build && mkdir -p /var/sig/tiles && chown -R www-data /var/sig/tiles
```

Comment:
Where do you want to push the tiles? We use this image in ECA project and we get some read-write right access issue in the past. @pvalsecc any though?

Reply:
If this image should have some specific thing for the ECA she shouldn't use a generic name...

Reply:
ECA seem to use the version 1.4, to have a smmoth transition we can do that only for the version 1.6 ...

Reply:
By the way what's the deference between the branch 1.6 and master?

Reply:
And 1.0, 1.2, 1.4...

Reply:
When we use something in a project, any changes made should not break it. And if so, we should be sure that thje changes needed oin the project are made. I am not against this changes, I want to be sure that this will not break my project. So if the /var/sig/tiles should be managed in the project side, we should change it in ECA.

So my question is where (or how) do you set the tiles directory? I will test an upgrade next week.

Finally, I think we should apply your change in all branch.

I am going to look the difference between btranches right now.

Reply:
there is something weird in the branches, mapcache version are not correct in all branches but master :/

Reply:
By the way, I think that we should have a Readme only int he master brances, and Docker files only in the version branches, see: https://github.com/camptocamp/docker-python-gis

Reply:
follow up the conf call:
- Yves update the mapcache_version in another PR
- Yves test the changes in internal project
- Stéphane and Patrick will check that the use of configuration image + "volumes from" + docker-entrypoint.d system is not an issue for long terme use